app/themes/default/views/layouts/default.html.slim

```diff
@@ -38,6 +38,13 @@ html
             li
               = link_to(t('layout.navbar.help'), '#')
             - if user_signed_in?
+              - if current_user.administrator?
+                li
+                  a.dropdown-toggle data-toggle="dropdown" = current_user.name
+                    b.caret
+                  ul.dropdown-menu
+                    li
+                      = link_to(t('layout.navbar.admin_panel'), admin_path)
```

Comment:
So there's some dissonance here. This is the global admin but actually it should check the instance permissions... interesting.